### DISCUSSION TITLE
release of 5.4

### DISCUSSION THREAD

**Discussion by wnunezc** [twbs/bootstrap#38829] [General]
Hello friends, were you expecting a date for the release of 5.4... or an approximation to it?

**Comment by julien-deramond** [comment]
Hello @arcanisgk 
Unfortunately, we don't know yet. We're currently trying to define the content of a potential v5.3.1.
Since we're all working on Bootstrap in our spare time, it's generally challenging to even have an approximation of the release dates.

**Reply by wnunezc** [reply]
I would even like to know if it will be ready before the end of the year, but it seems not, so I should ask for 5.3.1
